src/main/java/strategy/ReturnOperationHandler.java

```diff
@@ -0,0 +1,10 @@
+package strategy;
+
+import java.util.Map;
+
+public class ReturnOperationHandler implements OperationHandler {
+    @Override
+    public void apply(Map<String, Integer> inventory, String fruit, int quantity) {
+        inventory.put(fruit, inventory.getOrDefault(fruit, 0) + quantity);
+    }
```

Comment:
The `ReturnOperationHandler` class should not start with an empty line. According to the checklist, don't begin class or method implementation with an empty line.